.github/workflows/release.yml

```diff
@@ -0,0 +1,123 @@
+name: Release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build-release:
+    name: Build Release for ${{ matrix.arch_name }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    strategy:
+      matrix:
+        include:
+          - arch_name: x86_64
+            container: chocotechnologies/dmod:1.0.2
+            install_toolchain: ""
+          - arch_name: armv7-cortex-m4
+            container: chocotechnologies/dmod:1.0.2
+            install_toolchain: "apt-get update && apt-get install -y gcc-arm-none-eabi"
+          - arch_name: armv7-cortex-m7
+            container: chocotechnologies/dmod:1.0.2
+            install_toolchain: "apt-get update && apt-get install -y gcc-arm-none-eabi"
+    
+    container:
+      image: ${{ matrix.container }}
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      
+      - name: Install toolchain for ${{ matrix.arch_name }}
+        if: matrix.install_toolchain != ''
+        run: ${{ matrix.install_toolchain }}
+      
+      - name: Build dmffs for ${{ matrix.arch_name }}
+        run: |
+          set -e
+          mkdir -p build_${{ matrix.arch_name }}
+          cd build_${{ matrix.arch_name }}
+          
+          # First run cmake to fetch dependencies
+          cmake .. -DDMOD_MODE=DMOD_MODULE
+          
+          # Find dmod source directory and set DMOD_TOOLS if not x86_64
+          if [ "${{ matrix.arch_name }}" != "x86_64" ]; then
+            DMOD_SRC_DIR=$(find . -path "*/_deps/dmod-src" -type d | head -1)
+            
+            if [ -z "$DMOD_SRC_DIR" ]; then
+              echo "Error: Could not find dmod source directory"
+              exit 1
+            fi
+            
+            # Determine the correct tools config path based on architecture
+            case "${{ matrix.arch_name }}" in
+              armv7-cortex-m4)
```

Comment:
No to już jest jakiś cyrk... Ręcznie edytować listę architektur w dwóch różnych miejscach? To po chuj używasz tego matrix?